Misc/NEWS.d/next/Core_and_Builtins/2025-05-04-19-46-14.gh-issue-133346.nRXi4f.rst

```diff
@@ -0,0 +1 @@
+Added experimental color theming support to the ``_colorize`` module.
```

Comment:
No what's new entry?

Reply:
There is already a What's New entry with the syntax highlighting :D It's low-profile, sure.